region/info.json

```diff
@@ -5,5 +5,6 @@
     "description" : "Change the Discord server's region with a command.",
     "install_msg" : "Thanks for installing.",
     "tags" : ["voice region", "region"],
-    "disabled" : false
+    "disabled" : false,
+	"end_user_data_statement": "This cog does not persistently store data or metadata about users."
```

Comment:
Indentation is wonky on this, can you reformat the whole file to match your line or move your line in-line with the existing formatting please